ci/kind/kind-setup.sh

```diff
@@ -438,6 +483,9 @@ EOF
 }
 
 function destroy {
+  if [[ $FLEXIBLE_IPAM == true ]]; then
```

Comment:
I think typically we don't require the parameter used to setup the cluster to be specified when destroying it. The other cleanup function runs unconditionally.

Reply:
in case of ipam the pod routes cleanup dependent on cluster podCIDR. see:https://github.com/antrea-io/antrea/pull/5974#discussion_r1709328669

Reply:
How is it related to the condition check?

Reply:
according to:https://github.com/antrea-io/antrea/pull/5974#discussion_r1714728355, will rm it.